ng-dev/pr/common/checkout-pr.ts

```diff
@@ -109,6 +113,13 @@ export async function checkOutPullRequestLocally(
   try {
     // Fetch the branch at the commit of the PR, and check it out in a detached state.
     info(`Checking out PR #${prNumber} from ${fullHeadRef}`);
+    info(`To push to this PR branch:`);
+    info(
+      `  git push ${GITHUB_SSH_PREFIX}:${pr.headRef.repository.nameWithOwner} HEAD:${headRefName} -f`,
```

Comment:
Personally, I don't think we should provide the ssh prefixed version.  The majority of our team now uses https, and we already know that it works for our users as the tooling uses https for its own pushes it performs.  While the ssh is definitely valid, it won't work unless already set up where as the http one we know should work.

---

We don't  want to instruct users to use `-f` in this case, they should use `--force-with-lease`.  I think our best bet will actually be to leverage the information we already have in the shared checkout-pr.ts utils.  We can have the checkout function provide the command to push back to the branch and then use that here.  This would allow us to maintain this expected command in just one place.